src/util/psbt/mod.rs

```diff
@@ -91,6 +91,29 @@ impl PartiallySignedTransaction {
     }
 }
 
+#[cfg(feature = "base64")]
+mod _base64encoding {
+    use super::{PartiallySignedTransaction, Error};
+    use std::fmt::{Display, Formatter, self};
+    use std::str::FromStr;
+
+    impl Display for PartiallySignedTransaction {
+        fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+            use consensus::encode::serialize;
+            f.write_str(&::base64::encode(&serialize(self)))
```

Comment:
This allocation is avoidable by using [`base64::display::Base64Display`](https://docs.rs/base64-compat/1.0.0/base64/display/index.html). I'm not sure if it's worth the inconsistency with decoding, but wanted to point out that option.